cli/src/commands/subgraph/commands/publish.ts

```diff
@@ -95,7 +95,7 @@ export default (opts: BaseCommandOptions) => {
 
     switch (resp.response?.code) {
       case EnumStatusCode.OK: {
-        spinner.succeed('Subgraph published successfully.');
+        spinner.succeed(resp.response?.details || 'Subgraph published successfully.');
```

Comment:
The purpose of details was to communicate error details. We should not use it to communicate a state. Just add another property.

Reply:
Done